docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -65,6 +65,18 @@ def downstream_asset(upstream_asset):
 
 The [explicit dependencies](#explicit-dependencies) example covers an alternative way to specify asset dependencies without needing to match argument names to upstream asset names.
 
+### Using context in assets
+
+Since a software-defined asset contains an op, all of the typical functionality of an op (like the use of resources) is available to an asset.
+
+```python file=/concepts/assets/asset_w_context.py startafter=start_w_context endbefore=end_w_context
+@asset(required_resource_keys={"alerter"})
+def my_asset(context):
+    # creates an asset and sends an alert to notify of its creation
```

Comment:
It's typically the IOManager that does the actual asset creation - the function just returns the contents that the IOManager puts in persistent storage. A more common use case for using a resource inside the body of an asset could be fetching data from an external API.